### PR TITLE
Add ID translation for /docs/tutorials/kubernetes-basics/update/...

### PR DESCRIPTION
**Pull request by habibrosyad**
Translate pages under `/docs/tutorials/kubernetes-basics/update/...` as mentioned in #22296

### CODE REVIEW

File: content/id/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+            <div class="col-md-8">
+            <h3>Memperbarui suatu aplikasi</h3>
+
+            <p>Pengguna mengharapkan aplikasi tersedia sepanjang waktu dan pengembang bisa jadi diharapkan untuk men-<i>deploy</i> versi terbaru dari aplikasi tersebut beberapa kali dalam suatu waktu. Di Kubernetes hal ini dilakukan melalui pembaruan bertahap. <b>Pembaruan bertahap</b> memungkinkan pembaruan Deployment terjadi tanpa waktu henti dengan memperbarui instans Pod secara bertahap dengan versi yang baru. Pod yang baru akan dijadwalkan pada Node dengan sumber daya yang tersedia.</p>
+
+            <p>Dalam modul sebelumnya kita telah menskalakan aplikasi kita untuk menjalankan beberapa instans. Hal ini dibutuhkan untuk menjalankan pembaruan tanpa mempengaruhi ketersediaan aplikasi. Secara <i>default</i>, jumlah maksimum dari Pod lama yang akan digantikan (menjadi tidak tersedia) selama pembaruan dan jumlah maksimum Pod baru yang dapat dibuat adalah satu. Kedua opsi ini dapat dikonfigurasi dengan angka atau persentase (dari Pod). Di Kubernetes, setiap pembaruan diberi versi masing-masing dan suatu pembaruan Deployment dapat dikembalikan ke versi sebelumnya (yang stabil).</p>
```
Comment (ariscahyadi) [inline content/id/docs/tutorials/kubernetes-basics/update/update-intro.html:33]: “menskalakan” seems not official, please use “penyekalaan” https://kbbi.kemdikbud.go.id/entri/penyekalaan
use “bawaan” untuk “default”
```suggestion
            <p>Dalam modul sebelumnya kita telah melakukan penyekalaan aplikasi kita untuk menjalankan beberapa instans. Hal ini dibutuhkan untuk menjalankan pembaruan tanpa mempengaruhi ketersediaan aplikasi. Secara bawaan (<i>default</i>), jumlah maksimum dari Pod lama yang akan digantikan (menjadi tidak tersedia) selama pembaruan dan jumlah maksimum Pod baru yang dapat dibuat adalah satu. Kedua opsi ini dapat dikonfigurasi dengan angka atau persentase (dari Pod). Di Kubernetes, setiap pembaruan diberi versi masing-masing dan suatu pembaruan Deployment dapat dikembalikan ke versi sebelumnya (yang stabil).</p>
```

File: content/id/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_lined">
+                    <h3>Ringkasan:</h3>
+                    <ul>
+                        <li>Memperbarui aplikasi</li>
+                    </ul>
+                </div>
+                <div class="content__box content__box_fill">
+                    <p><i>Pembaruan bertahap memungkinkan pembaruan Deployment terjadi tanpa waktu henti dengan memperbarui instans Pod secara bertahap dengan versi yang baru.</i></p>
```
Comment (ariscahyadi) [inline content/id/docs/tutorials/kubernetes-basics/update/update-intro.html:44]: nit!
```suggestion
                    <p><i>Pembaruan bertahap memungkinkan pembaruan Deployment terjadi tanpa berhenti dengan memperbarui instans Pod secara bertahap dengan versi yang baru.</i></p>
```

File: content/id/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+                    </div>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+
+                <p>Seperti penskalaan aplikasi, jika suatu Deployment terekspos secara publik, maka Service akan me-<i>load-balance</i> trafik hanya ke Pod yang tersedia saat pembaruan berlangsung. Pod yang tersedia adalah instans yang dapat diakses oleh pengguna aplikasi.</p>
```
Comment (ariscahyadi) [inline content/id/docs/tutorials/kubernetes-basics/update/update-intro.html:98]: official word is “penyekalaan” https://kbbi.kemdikbud.go.id/entri/penyekalaan
```suggestion
                <p>Seperti penyekalaan aplikasi, jika suatu Deployment terekspos secara publik, maka Service akan menyeimbangkan beban trafik hanya ke Pod yang tersedia saat pembaruan berlangsung. Pod yang tersedia adalah instans yang dapat diakses oleh pengguna aplikasi.</p>
```

File: content/id/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+                <p>Pembaruan bertahap memungkinan dilakukannya aktivitas-aktivitas berikut:</p>
+                <ul>
+                    <li>Mempromosikan suatu aplikasi dari satu <i>environment</i> yang ke <i>environment</i> yang lain (via pembaruan kontainer <i>image</i>)</li>
+                    <li>Kembali (<i>rollback</i>) ke versi-versi sebelumnya</li>
+                    <li>Continuous Integration dan Continuous Delivery (CI/CD) dari aplikasi tanpa waktu henti</li>
```
Comment (ariscahyadi) [inline content/id/docs/tutorials/kubernetes-basics/update/update-intro.html:104]:
```suggestion
                    <li>Continuous Integration dan Continuous Delivery (CI/CD) dari aplikasi tanpa henti</li>
```

File: content/id/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+                </ul>
+
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_fill">
+                    <p><i>Jika suatu Deployment terekspos secara publik, maka Service akan me-<i>load-balance</i> trafik hanya ke Pod yang tersedia saat pembaruan berlangsung.</i></p>
```
Comment (ariscahyadi) [inline content/id/docs/tutorials/kubernetes-basics/update/update-intro.html:110]:
```suggestion
                    <p><i>Jika suatu Deployment terekspos secara publik, maka Service akan menyeimbangkan beban trafik hanya ke Pod yang tersedia saat pembaruan berlangsung.</i></p>
```

File: content/id/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+        <div class="row">
+            <div class="col-md-8">
+                <p>Pada tutorial interaktif berikut ini, kita akan mencoba untuk memperbarui aplikasi kita ke versi terbaru dan kemudian melakukan <i>rollback</i>.</p>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-12">
+                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/update/update-interactive/" role="button">Mulai Tutorial Interaktif <span class="btn__next">›</span></a>
```
Comment (ariscahyadi) [inline content/id/docs/tutorials/kubernetes-basics/update/update-intro.html:126]: change to ID doc if available.
```suggestion
                <a class="btn btn-lg btn-success" href="/id/docs/tutorials/kubernetes-basics/update/update-interactive/" role="button">Mulai Tutorial Interaktif <span class="btn__next">›</span></a>
```

**Review by ariscahyadi — COMMENTED** [review]
Thanks @habibrosyad for your contribution. Please small comments from me.

File: content/id/docs/tutorials/kubernetes-basics/update/update-intro.html (hunk 1 above)
Comment (habibrosyad) [reply content/id/docs/tutorials/kubernetes-basics/update/update-intro.html]: Done in 11ec389

File: content/id/docs/tutorials/kubernetes-basics/update/update-intro.html (hunk 2 above)
Comment (habibrosyad) [reply content/id/docs/tutorials/kubernetes-basics/update/update-intro.html]: ditto

File: content/id/docs/tutorials/kubernetes-basics/update/update-intro.html (hunk 3 above)
Comment (habibrosyad) [reply content/id/docs/tutorials/kubernetes-basics/update/update-intro.html]: ditto

File: content/id/docs/tutorials/kubernetes-basics/update/update-intro.html (hunk 4 above)
Comment (habibrosyad) [reply content/id/docs/tutorials/kubernetes-basics/update/update-intro.html]: ditto

**Comment by habibrosyad** [comment]
All comments have been addressed, could you check again @ariscahyadi? Thanks for reviewing, got some new for translation vocabularies (kinda familiar with the terms but don't know how to translate it to Bahasa) 👍

**Comment by habibrosyad** [comment]
Some previews:
- https://deploy-preview-23907--kubernetes-io-master-staging.netlify.app/id/docs/tutorials/kubernetes-basics/update
- https://deploy-preview-23907--kubernetes-io-master-staging.netlify.app/id/docs/tutorials/kubernetes-basics/update/update-intro/
- https://deploy-preview-23907--kubernetes-io-master-staging.netlify.app/id/docs/tutorials/kubernetes-basics/update/update-interactive/

**Comment by ariscahyadi** [comment]
I think it is a good to go. Thank you @habibrosyad for your contribution.

/lgtm

**Comment by girikuncoro** [comment]
Thanks for your contribution @habibrosyad, hope you learned something from localizing stuffs! 🎉 

/lgtm
/approve

**Pull request merged**